include/battle.h

```diff
@@ -184,6 +184,7 @@ struct SpecialStatus
     u8 physicalBattlerId;
     u8 specialBattlerId;
     u8 changedStatsBattlerId; // Battler that was responsible for the latest stat change. Can be self.
+    bool8 affectionEndured:1;
```

Comment:
Maybe put it after neutralizingGasRemoved, and change bool8 to u8 as with different bitfields.

Reply:
> Maybe put it after neutralizingGasRemoved
Uh, sure I guess. Why though? 🤔 

> and change bool8 to u8 as with different bitfields.
It's used only to signal `TRUE`/`FALSE` states though 🤔 ...

Reply:
For consistency with other bitfields 😄

Reply:
Alrighty.